inc/newslettersubscriber.class.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ LICENSE
```

Comment:
I know, I have to port to the demo plugin the work done on the main one. This will be a dedicated and separate task